lighthouse-core/lib/network-recorder.js

```diff
@@ -168,15 +165,17 @@ class NetworkRecorder extends EventEmitter {
   }
 
   /**
-   * Construct network records from a log of devtools protocol messages.
-   * @param {!DevtoolsLog} log
+   * Construct network records from a log of devtools protocol messages. If a
+   * driver is provided, it will be used for request content lookups.
+   * @param {!DevtoolsLog} devtoolsLog
+   * @param {!Driver=} driver
```

Comment:
IMO, audits shouldn't need to live request content if gatherers didn't save them into a separate artifact meaning it can just be a method on a driver to `getResourceContent` and pass in a network record id, the construction of network records is not capable of reconstructing the content anyway